### ISSUE TITLE
Game crashes when near tower

### ISSUE THREAD

**Issue by dianchia** [dianchia/DefenseTower#7]

**Comment by Diseenith** [comment]
I saw your request on steam for a crash log, it happens when I mouse over the tower in any way. I logged in and forced the crash so I could provide you a log.

[2022-09-28 18:42:00] SYSTEM INFO:
[2022-09-28 18:42:00] 	Username: Anthr
[2022-09-28 18:42:01] 	OS: Microsoft Windows 10 (Home) build 19044
[2022-09-28 18:42:01] 	CPU: Intel(R) Core(TM) i5-10400F CPU @ 2.90GHz
[2022-09-28 18:42:01] 	OS arch: amd64
[2022-09-28 18:42:01] 	Memory total: 34.29 GB
[2022-09-28 18:42:01] 	Memory max: 4.29 GB
[2022-09-28 18:42:01] 	Java path: X:\SteamLibrary\steamapps\common\Necesse\jre
[2022-09-28 18:42:01] 	Java version: 1.8.0_202
[2022-09-28 18:42:01] 	JVM arguments:
[2022-09-28 18:42:01] 		-Xms512m
[2022-09-28 18:42:01] 		-Xmx4G
[2022-09-28 18:42:01] 		-XX:+UnlockExperimentalVMOptions
[2022-09-28 18:42:01] 		-XX:+UseG1GC
[2022-09-28 18:42:01] 		-XX:G1NewSizePercent=20
[2022-09-28 18:42:01] 		-XX:G1ReservePercent=20
[2022-09-28 18:42:01] 		-XX:MaxGCPauseMillis=50
[2022-09-28 18:42:01] 		-XX:G1HeapRegionSize=32M
[2022-09-28 18:42:01] 	LWJGL version: 3.3.1 build 7
[2022-09-28 18:42:01] 	Root path: X:\SteamLibrary\steamapps\common\Necesse\
[2022-09-28 18:42:01] 	AppData path: C:\Users\Anthr\AppData\Roaming\Necesse\
[2022-09-28 18:42:01] 	Working dir: X:\SteamLibrary\steamapps\common\Necesse
[2022-09-28 18:42:01] 	Natives path: INTERNAL
[2022-09-28 18:42:02] GRAPHICS INFO:
[2022-09-28 18:42:02] 	Graphics card: NVIDIA Corporation, NVIDIA GeForce RTX 3060/PCIe/SSE2
[2022-09-28 18:42:02] 	OpenGL version: 4.6.0 NVIDIA 516.94
[2022-09-28 18:42:02] 	Display 0: ASUS VP278 (AEC, 27ae)
[2022-09-28 18:42:02] 	Display 1: XF243Y P (ACB, 847)
[2022-09-28 18:42:02] Started client on version 0.21.25 build 9593077 with authentication 76561198024267707.
[2022-09-28 18:42:04] Found 3 subscribed items from Steam Workshop
[2022-09-28 18:42:04] Found mod: Increased Stack Size (dianchia.increasedstacksize, 1.0.4) from STEAM_WORKSHOP
[2022-09-28 18:42:04] Found mod: Faster Path (dianchia.fasterpath, 1.0.2) from STEAM_WORKSHOP
[2022-09-28 18:42:04] Found mod: Defense Tower (dianchia.defensetower, 1.1.0) from STEAM_WORKSHOP
[2022-09-28 18:42:05] dianchia.increasedstacksize transformed necesse.inventory.item.Item.getStackSize()
[2022-09-28 18:42:05] dianchia.increasedstacksize transformed necesse.inventory.item.Item.inventoryAddItem(Level, PlayerMob, InventoryItem, InventoryItem, String, boolean, int)
[2022-09-28 18:42:05] dianchia.increasedstacksize transformed necesse.engine.world.WorldSettings.applyContentPacket(PacketReader)
[2022-09-28 18:42:05] dianchia.increasedstacksize transformed necesse.engine.world.WorldSettings.getSaveScript()
[2022-09-28 18:42:05] dianchia.increasedstacksize transformed necesse.engine.world.WorldSettings.loadSaveScript(LoadData)
[2022-09-28 18:42:05] dianchia.increasedstacksize transformed necesse.engine.world.WorldSettings.setupContentPacket(PacketWriter)
[2022-09-28 18:42:05] dianchia.fasterpath transformed necesse.level.gameTile.PathTiledTile.getItemTooltips(InventoryItem, PlayerMob)
[2022-09-28 18:42:05] dianchia.fasterpath transformed necesse.level.gameTile.PathTiledTile.getSpeedModifier(Mob)
[2022-09-28 18:42:05] en.lang: itemtooltip.stonepathtip localisation overwriting by mod dianchia.fasterpath
[2022-09-28 18:42:05] Checking for backward compatibility...
[2022-09-28 18:42:05] Initializing FasterPath
[2022-09-28 18:42:05] Loading config...
[2022-09-28 18:42:05] Initialization completed! Path is now faster by 2.0x!
[2022-09-28 18:42:12] Loaded Increased Stack Size mod... Enjoy!
[2022-09-28 18:42:46] Started singleplayer server on world Dizzy.zip, game version 0.21.25
[2022-09-28 18:42:46] Found 1 saved players.
[2022-09-28 18:42:46] Client "Dizzy Bat" connected on slot 1/1.
[2022-09-28 18:42:46] Loading map for player: AUTH:76561198024267707
[2022-09-28 18:42:46] Loaded player: 76561198024267707
[2022-09-28 18:42:46] Adding stack size to packet. Stack size: 5
[2022-09-28 18:42:46] Chat > Dizzy Bat is connecting!
[2022-09-28 18:42:46] New stack size from packet = 5
[2022-09-28 18:42:48] Chat > Dizzy Bat has joined!
[2022-09-28 18:43:31] Dizzy Bat is now AFK
[2022-09-28 18:44:15] Dizzy Bat is no longer AFK
[2022-09-28 18:45:57] Chat > §aAlexander the Pawnbroker is arriving at Desert City
[2022-09-28 18:46:30] (ERR) Error in ticking:
[2022-09-28 18:46:30] (ERR) java.lang.NoSuchMethodError: necesse.engine.Screen.addTooltip(Lnecesse/gfx/gameTooltips/GameTooltips;)V
[2022-09-28 18:46:30] (ERR) 	at DefenseTower.objectEntity.DefenseTowerEntity.onMouseHover(DefenseTowerEntity.java:123)
[2022-09-28 18:46:30] (ERR) 	at DefenseTower.gameObject.DefenseTowerObject2.onMouseHover(DefenseTowerObject2.java:44)
[2022-09-28 18:46:30] (ERR) 	at necesse.level.maps.LevelObject.onMouseHover(LevelObject.java:118)
[2022-09-28 18:46:30] (ERR) 	at necesse.gfx.ui.HUD.draw(HUD.java:320)
[2022-09-28 18:46:30] (ERR) 	at necesse.gfx.drawables.LevelDrawUtils$2.draw(LevelDrawUtils.java:548)
[2022-09-28 18:46:30] (ERR) 	at necesse.gfx.drawables.LevelDrawUtils.lambda$null$56(LevelDrawUtils.java:641)
[2022-09-28 18:46:30] (ERR) 	at java.util.ArrayList.forEach(ArrayList.java:1257)
[2022-09-28 18:46:30] (ERR) 	at java.util.Collections$SynchronizedCollection.forEach(Collections.java:2062)
[2022-09-28 18:46:30] (ERR) 	at necesse.gfx.drawables.LevelDrawUtils.lambda$drawLastHudDrawables$57(LevelDrawUtils.java:641)
[2022-09-28 18:46:30] (ERR) 	at necesse.engine.tickManager.PerformanceTimerManager.recordPerformance(PerformanceTimerManager.java:105)
